src/pages/ShoppingCart.page.js

```diff
@@ -1,6 +1,7 @@
-const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+import { getItemsListData } from '../HelperFunctions.util';
+import { InventoryPage } from './Inventory.page';
 
-export class ShopingCartPage extends BaseSwagLabPage {
+export class ShoppingCartPage extends InventoryPage {
```

Comment:
is it correct to extend the Inventory page?

Reply:
seems like I tried something by using methods of an extendable class, but failed. so it's not necessary